tasmoadmin/config.json

```diff
@@ -16,20 +16,23 @@
   "boot": "auto",
   "hassio_api": true,
   "hassio_role": "default",
+  "ingress": true,
+  "ingress_port": 8099,
   "map": [
     "ssl"
   ],
   "ports": {
-    "9541/tcp": 9541
+    "9541/tcp": null
   },
   "ports_description": {
-    "9541/tcp": "TasmoAdmin web interface"
+    "9541/tcp": "TasmoAdmin web interface (Not required for Hass.io Ingress)"
   },
   "options": {
     "ssl": true,
```

Comment:
should we change this to `false` by default? Not sure what the common setup is, but setup doesn't have a certfile/keyfile by default, so this thing actually fails to start-up with the defaults.

Reply:
No. It should be true, if you like to run things insecure, you have to put in effort.